### DISCUSSION TITLE
How to apply it to exist prisma`^4.13.0` ?

### DISCUSSION THREAD

**Discussion by toknT** [keystonejs/keystone#8757] [Questions]
It can generate all admin panels really look good. But after add `Admin` table to exist project and copy the `schema.prisma` from our existed project `nestjs`+`prisma`(`"@nestjs/core": "^9.0.0"` ,`"prisma": "^4.13.0"`) get prisma not update error

```sh
npx keystone prisma db pull  
npx keystone prisma generate   
# Your Prisma schema is not up to date
```

**Comment by dcousens** [comment]
Did you try
```bash
keystone build
```

**Reply by toknT** [reply]
`keystone build` overwrite the `schema.prisma` not worked. So I need write `schema.ts` manually by hand , becasue only 6 table need in admin panel so I run it with `--no-db-push`.

**Comment by toknT** [comment]
From https://stackoverflow.com/questions/71228292/generate-keystone-js-6-schema-from-modified-prisma-schema I manually create a `schema.prisma` file and run `--no-db-push`(I did not write all tables in schema.ts)
